eeps/eep-0054.md

```diff
@@ -206,10 +202,23 @@ be implemented:
     do_format_error(_, _, _) ->
         #{}.
 
-The `register/2` BIF will provide specific error reasons for two of
-the possible failure reasons.  If the reason is not one of the two,
-`format_error/4` will figure out the other reasons based on the
-arguments.
+Note that different strategies for different BIFS are used for
+determining the extended error information:
+
+* First the `system_limit` exception is handled (regardless of which
+  BIF that was called).  No extended error information is returned,
```

Comment:
I believe `that` can be suppressed here.
or you could also write "regardless of the BIF that was called"